Makefile

```diff
@@ -40,3 +29,8 @@ update-docs:
 .PHONY: update-plugin-sdk
 update-plugin-sdk:
 	./scripts/update-plugin-sdk.sh
+
+
+.PHONY: list
+list:
+	@grep '^[^#[:space:].].*$$' Makefile | sed 's/\(.*\):/"\1",/' | tr -d '\n' | sed 's/,$$//g'
```

Comment:
I know this is for CI only, but it doesn't work on mac unless you do `grep -E` at the start. It also seems to find some targets (from targets that invoke other targets) multiple times.

```
grep -E '^[^#[:space:].].*$$' Makefile | sed 's/\(.*\):/"\1",/' | tr -d '\n' | sed 's/,$$//g'
"gen-mocks","test","install-tools","install-hooks","gen-docs","lint","gen-code","gen", gen-code gen-mocks gen-docs
```